### PR TITLE
Bug fixes + Multi S3ImageUploadModelField support

### PR DESCRIPTION
**Pull request by beshrkayali**

### CODE REVIEW

File: djpressor/s3.py [hunk 1]
```diff
@@ -1,8 +1,39 @@
...
+            sock = socket.create_connection(
+                (self.host, self.port), self.timeout)
+        else:
+            sock = socket.create_connection(
+                (self.host, self.port))
+
+        if re.match(".*\.s3.*\.amazonaws\.com", self.host):
+            patched_host = ".".join(self.host.rsplit(".", 4)[1:])
+        self.sock = self.ssl_ctx.wrap_socket(
+            sock, server_hostname=patched_host)
```
Comment (flavianmissi) [inline djpressor/s3.py:35]: What is the python version targeted for this fix? `2.7.10`?

**Comment by flavianmissi** [comment]
@beshrkayali are the tests gonna come in a separate PR?

File: djpressor/s3.py (hunk 1 above)
Comment (beshrkayali) [inline djpressor/s3.py:35]: @flaviamissi Yeah >= 2.7.9 but let's hold off on merging for a sec.

**Comment by beshrkayali** [comment]
@flaviamissi regarding tests, yeah it's ok, no major areas need extra coverage.. i have another local branch i'm working on with extended cases but it's not done yet, i'll push it along with this one...

**Pull request merged**